src/test/java/org/schemaspy/model/TableColumnTest.java

```diff
@@ -0,0 +1,47 @@
+package org.schemaspy.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+class TableColumnTest {
+
+    @Test
+    void noImplied() {
```

Comment:
Change the test names to be self-documenting. A tip could be to describe the object's expected behaviour.

Reply:
I'll fix this